unittest/case.py:678: DeprecationWarning: It is deprecated to return a value that is not None from a test case (<bound method RouterTestBadConfiguration.test_skmanage_query of <tests.system_tests_bad_configuration.RouterTestBadConfiguration testMethod=test_skmanage_query>>)

```
2023-03-24T18:23:16.1233035Z 40: [1m[31mE   AssertionError: 'AttributeError: Router id attribute containing character' not found in 'skrouterd: error while loading shared libraries: libclang_rt.asan-x86_64.so: cannot open shared object file: No such file or directory\n' : skrouterd: error while loading shared libraries: libclang_rt.asan-x86_64.so: cannot open shared object file: No such file or directory[0m
2023-03-24T18:23:16.1233908Z 40: [33m======================================================================= warnings summary =======================================================================[0m
2023-03-24T18:23:16.1234255Z 40: ::RouterTestBadConfiguration::test_skmanage_query
2023-03-24T18:23:16.1234910Z 40:   /usr/lib64/python3.11/unittest/case.py:678: DeprecationWarning: It is deprecated to return a value that is not None from a test case (<bound method RouterTestBadConfiguration.test_skmanage_query of <tests.system_tests_bad_configuration.RouterTestBadConfiguration testMethod=test_skmanage_query>>)
2023-03-24T18:23:16.1235493Z 40:     return self.run(*args, **kwds)
```

https://github.com/jiridanek/skupper-router/actions/runs/4514090401/jobs/7949756317#step:32:22356

--- a/tests/system_tests_bad_configuration.py
+++ b/tests/system_tests_bad_configuration.py
@@ -141,11 +141,10 @@ class RouterTestBadConfiguration(TestCase):
         """
         self.assertTrue(self.error_caught)
 
-    def test_skmanage_query(self):
+    def test_zzz_skmanage_query(self):
         """
         Attempts to query the router after the error (or timeout) has occurred.
         It expects a successful query response to be returned by the router.
-        :return:
         """
         p = self.popen(
             ['skmanage', '-b', self.address(), 'query', '--type=router', '--timeout', str(TIMEOUT)],
@@ -157,7 +156,6 @@ class RouterTestBadConfiguration(TestCase):
             p.kill()
             out, _ = p.communicate(timeout=TIMEOUT)
             raise Exception("%s\n%s" % (e, out)) from e
-        return out
 
 
 class RouterTestIdFailChar(TestCase):

